frame/ethereum/Cargo.toml

```diff
@@ -24,13 +24,13 @@ ethereum = { version = "0.9.0", default-features = false, features = ["with-code
 ethereum-types = { version = "0.12", default-features = false }
 rlp = { version = "0.5", default-features = false }
 sha3 = { version = "0.8", default-features = false }
-libsecp256k1 = { version = "0.5", default-features = false, features = ["static-context", "hmac"] }
 fp-consensus = { version = "2.0.0-dev", path = "../../primitives/consensus", default-features = false }
 fp-rpc = { version = "3.0.0-dev", path = "../../primitives/rpc", default-features = false }
 fp-storage = { version = "2.0.0", path = "../../primitives/storage", default-features = false}
 
 [dev-dependencies]
 sp-core = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate" }
+libsecp256k1 = { version = "0.5", default-features = false, features = ["static-context", "hmac"] }
```

Comment:
Set `resolver = "2"` and you can remove `default-features = false`.

Reply:
I believe I've done as you asked, but please review carefully, because I was not familiar with `resolver = "2"` until today. I read https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions.

Reply:
Running the *pallet-ethereum* unit test cases without setting resolver = "2" and default-features = false also works.
Curious what's the difference between set `resolver = "2"` or not.